### PR TITLE
Allow `FlxCamera#followLerp` to be used with `NO_DEAD_ZONE`

### PR DESCRIPTION
**Pull request by DalekCraft2**

### CODE REVIEW

File: flixel/FlxCamera.hx [hunk 1]
```diff
@@ -1178,7 +1178,8 @@ class FlxCamera extends FlxBasic
 		{
 			target.getMidpoint(_point);
 			_point.addPoint(targetOffset);
-			focusOn(_point);
+			_scrollTarget.set(_point.x - width * 0.5, _point.y - height * 0.5);
+			_point.putWeak();
```
Comment (Geokureli) [inline flixel/FlxCamera.hx:1182]: _point is an instance var, it does not get put and will never be weak

**Review by Geokureli — CHANGES_REQUESTED** [review]
CI is failing for what seems like network issues, remove the putWeak and I'll re-run CI as needed

**Review by Geokureli — APPROVED** [review]

**Comment by DalekCraft2** [comment]
I was having connection issues when running the `haxelib` command in command prompt around the same time as you were doing the CIs, but it's running fine now, so maybe the CIs will work if they are re-run now.

**Comment by Geokureli** [comment]
thanks!

**Pull request merged**